galaxy_ng/tests/integration/api/test_container_push_update.py

```diff
@@ -0,0 +1,55 @@
+
+"""Tests related to container push update.
+
+See: https://issues.redhat.com/browse/AAH-2327
+"""
+import subprocess
+import time
+
+import pytest
+
+from galaxy_ng.tests.integration.utils import get_client
+
+
+@pytest.mark.parametrize(
+    "require_auth",
+    [
+        True,
+        False,
+    ],
+)
+@pytest.mark.standalone_only
+def test_can_update_container_push(ansible_config, require_auth):
+    # Pull alpine image
+    subprocess.check_call(["docker", "pull", "alpine"])
```

Comment:
Can you switch this to podman? Podman is easier to run in container.

Reply:
This is how all other container tests are written, so lets have a separate PR to switch to podman, I guess it will need to be installed/setup on the testing pipeline.